capture_packets.py

```diff
@@ -1,17 +1,19 @@
 #! /usr/bin/env python3
 from scapy.all import sniff
-from scapy.utils import raw
-import re, sys, json, time
-from privacy_analysis.privacy_class import Privacy_Analysis
-
-# Filepath to the config file to pull the MAC addresses from
-config_file_path = "config.json"
-
-# Base location of where packets will be stored
-packet_base_location = "../packets_captured"
+from cids_main import run_config
+import re, sys
+from src.privacy_analysis.packet_analysis.packet_privacy_port import PacketPrivacyPort
+from src.privacy_analysis.system_analysis.system_privacy_dropbear_config import SystemPrivacyDropbearConfig
+from src.privacy_analysis.system_analysis.system_privacy_encryption import SystemPrivacyEncryption
+from src.privacy_analysis.system_analysis.system_privacy_package_upgrades import SystemPrivacyPackageUpgrades
+from src.privacy_analysis.system_analysis.system_privacy_root_password import SystemPrivacyRootPassword
+
+rules_packet_privacy = [PacketPrivacyPort()]
+rules_system_privacy = [SystemPrivacyDropbearConfig(), SystemPrivacyEncryption(), SystemPrivacyPackageUpgrades(), SystemPrivacyRootPassword()]
+rules_scanning_privacy = []
 
 # Number of packets to capture, 0 is infinite
-num_packets = 2
+num_packets = 1
```

Comment:
If we want the IDS to run continuously, should this value be 0?

Reply:
Eventually, yes. It's not necessary now while we're testing